tool/gravity/cli/config.go

```diff
@@ -499,6 +502,28 @@ func (i *InstallConfig) RunLocalChecks() error {
 	}))
 }
 
+// BootstrapSELinux configures SELinux on a node prior to installation
+func (i *InstallConfig) BootstrapSELinux(ctx context.Context, printer utils.Printer) error {
+	if !i.SELinux || i.FromService || i.Mode == constants.InstallModeInteractive || i.Remote {
+		return nil
+	}
+	metadata, err := monitoring.GetOSRelease()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if !selinux.GetEnabled() {
+		i.WithField(trace.Component, "selinux").Info("SELinux not enabled on host.")
+		i.SELinux = false
+	} else if !libselinux.IsSystemSupported(metadata.ID) {
+		i.WithField("id", metadata.ID).Info("Distribution not supported.")
```

Comment:
Should this also use "component: selinux" field? Otherwise the logs will just say "distribution not supported" without any indication this is only SELinux-specific.